backend/src/main/java/gov/cdc/usds/simplereport/api/UploadScalarType.java

```diff
@@ -0,0 +1,16 @@
+package gov.cdc.usds.simplereport.api;
+
+import graphql.schema.GraphQLScalarType;
+import graphql.kickstart.servlet.apollo.ApolloScalars;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class UploadScalarType  {
```

Comment:
This should live in the general `simplereport.config` package, I think?

Reply:
I wasn't sure on where to place this. I imagine as we make our API more robust we are going to be adding more custom `ScalarTypes` and they do seem to be part of the `api` logic. Hoping as we add more it becomes clearer